src/Components/Publishing/Video/VideoAbout.tsx

```diff
@@ -1,84 +1,79 @@
-import { Box, color, Flex, Sans } from "@artsy/palette"
-import React, { Component } from "react"
+import { Box, Flex, Sans } from "@artsy/palette"
+import React from "react"
 import styled from "styled-components"
 
 import { ShareDate } from "Components/Publishing/Byline/ShareDate"
 import { StyledText } from "Components/Publishing/Sections/StyledText"
 import { Text } from "Components/Publishing/Sections/Text"
+import { ArticleData } from "Components/Publishing/Typings"
 import { Media } from "Utils/Responsive"
 
 export interface VideoAboutProps {
-  article: any
+  article: ArticleData
   color?: string
   editDescription?: any
   editCredits?: any
 }
 
-export class VideoAbout extends Component<VideoAboutProps, null> {
-  static defaultProps = {
-    color: color("black100"),
-  }
-
-  render() {
-    const { article, editCredits, editDescription } = this.props
-    const {
-      media: { credits, description },
-    } = article
+export const VideoAbout: React.SFC<VideoAboutProps> = ({
+  article,
+  color = "black",
+  editCredits,
+  editDescription,
+}) => {
+  const {
+    media: { credits, description },
+  } = article
 
-    return (
-      <VideoAboutContainer
-        maxWidth={1200}
-        mx="auto"
-        flexDirection={["column-reverse", "column-reverse", "row", "row"]}
+  return (
+    <VideoAboutContainer
+      maxWidth={1200}
+      mx="auto"
+      flexDirection={["column-reverse", "column-reverse", "row", "row"]}
+    >
+      <Credits
+        width={[1, 1, 1 / 3, 1 / 3]}
```

Comment:
This can be simplified to 
```
[1, 1, 1 / 3]
```
since it will automatically apply the last breakpoint upwards

Reply:
This didn't seems to work as expected when it was implemented a few weeks ago, maybe it has changed?

Reply:
ahh, it didn't? I haven't messed with it in a while. All good